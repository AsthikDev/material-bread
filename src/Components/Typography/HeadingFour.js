import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { StyleSheet } from 'react-native';
import BaseText from './BaseText';

const styles = StyleSheet.create({
  headingThree: {
    fontSize: 34,
    lineHeight: 34,
  },
});

export default class HeadingFour extends Component {
  static propTypes = {
    children: PropTypes.node,
  };

  render() {
    return (
      <BaseText typographyStyles={styles.headingThree} {...this.props}>
        {this.props.children}
      </BaseText>
    );
  }
}
