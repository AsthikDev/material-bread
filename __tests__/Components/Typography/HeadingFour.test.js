import React from 'react';
import HeadingFour from '../../../src/Components/Typography/HeadingFour';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<HeadingFour />).toJSON();
  expect(tree).toMatchSnapshot();
});
