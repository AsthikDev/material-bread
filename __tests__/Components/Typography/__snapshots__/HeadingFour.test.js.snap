// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "rgba(0, 0, 0, 0.87)",
      "fontSize": 34,
      "lineHeight": 34,
    }
  }
  typographyStyles={
    Object {
      "fontSize": 34,
      "lineHeight": 34,
    }
  }
/>
`;
